test/govm-start-common.sh

```diff
@@ -0,0 +1,211 @@
+. $(dirname $0)/test-config.sh
+NUM_NODES=3
+# override TEST_IP_ADDR that comes from test-config.sh (as clear-specific value) with govm value
+TEST_IP_ADDR=172.17.0
+NO_PROXY="$NO_PROXY,${TEST_IP_ADDR}.0/24,10.96.0.0/12,10.244.0.0/16"
```

Comment:
```
test/govm-start-common.sh: line 5: NO_PROXY: unbound variable
```

Reply:
uh, right. all my dev was behind proxy, but I should have been verified that its runnable without. will fix soon

Reply:
pushed new version, should work better with undefined proxy values